get-module-state.ts not working

When agent is build from the root directory, with node v16.17.0, get-module-state.ts produces:
```
$./get-module-state.ts
TypeError: semver.gte is not a function
    at isDeviceProtocolSupportGitInfo (file:///opt/agent/packages/uhk-common/src/util/is-device-protocol-support-git-info.ts:6:19)
    at UhkOperations.getRightModuleVersionInfo (file:///opt/agent/packages/uhk-usb/src/uhk-operations.ts:468:13)
    at async file:///opt/agent/packages/usb/get-module-state.ts:14:33
```

When `npm ci` is ran in `agent/packages/usb` (as its readme states it should be):
```
agent/packages/usb$ npm ci
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/kboot - Not found
npm ERR! 404
npm ERR! 404  'kboot@0.0.0' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/me/.npm/_logs/2022-09-07T09_59_11_606Z-debug-0.log
```

Also, root README states that node 12 is required, however in practice (if node 12 is actually used) build complains that it wants node 16.


//edit: fixed the kboot output.

--- a/packages/uhk-common/src/util/is-device-protocol-support-git-info.ts
+++ b/packages/uhk-common/src/util/is-device-protocol-support-git-info.ts
@@ -1,4 +1,4 @@
-import * as semver from 'semver';
+import semver from 'semver';
 
 const DEVICE_PROTOCOL_VERSION_THAT_SUPPORT_GIT_INFO = '4.8.0';
 

